docs/user-guide/configuration/annotations.md

```diff
@@ -61,3 +61,4 @@ Pipeline-level annotations are used to modify the properties of the entire pipel
  | Annotation | Values | Description |
  |------------|--------|-------------|
  | screwdriver.cd/restrictPR | `none` / `all` / `fork` / `branch` | Prevent PR jobs from running. `none` means no restriction. `all` means all PR jobs will not run. `fork` means PRs from forked repos will not run. `branch` means PRs from branch will not run.|
+ | screwdriver.cd/chainPR    | `false` / `true` | Default value is `false`. When value is `false` a Pull Request will run only those jobs which has `~pr` in `requires`. When `true` pull request workflow will behave like regular commit workflow. Workflow will start at jobs which has `~pr` in requires and will also trigger any downstream jobs in the workflow.|
```

Comment:
there's a lot of use of the word "workflow" here
seems kind of redundant without really giving the user any more information

Reply:
added link to workflow

Reply:
"those jobs which **has**" ---> **have**

For second part, it will be simpler to say
"When `true`, it will run jobs which have `~pr` in `requires` and also trigger their downstream jobs."